demo/capture_Snakefile.smk

```diff
@@ -25,75 +27,77 @@ subworkflow reference_files:
 
 
 # Load module-specific configuration
+configfile: "../modules/slms_3/1.0/config/default.yaml"
+configfile: "../modules/sage/1.0/config/default.yaml"
 configfile: "../modules/utils/2.1/config/default.yaml"
 configfile: "../modules/picard_qc/1.0/config/default.yaml"
-configfile: "../modules/salmon/1.1/config/default.yaml"
 configfile: "../modules/bam2fastq/1.2/config/default.yaml"
-configfile: "../modules/star/1.4/config/default.yaml"
 configfile: "../modules/manta/2.3/config/default.yaml"
 configfile: "../modules/gridss/1.1/config/default.yaml"
-configfile: "../modules/vcf2maf/1.2/config/default.yaml"
+#configfile: "../modules/vcf2maf/1.2/config/default.yaml"
 configfile: "../modules/sequenza/1.4/config/default.yaml"
 configfile: "../modules/strelka/1.1/config/default.yaml"
 configfile: "../modules/bwa_mem/1.1/config/default.yaml"
-configfile: "../modules/controlfreec/1.1/config/default.yaml"
 configfile: "../modules/lofreq/1.0/config/default.yaml"
 configfile: "../modules/starfish/2.0/config/default.yaml"
-configfile: "../modules/sage/1.0/config/default.yaml"
-configfile: "../modules/slms_3/1.0/config/default.yaml"
-
+configfile: "../modules/battenberg/1.1/config/default.yaml"
+configfile: "../modules/mutect2/2.0/config/default.yaml"
+configfile: "../modules/varscan/1.1/config/default.yaml"
+configfile: "../modules/liftover/1.2/config/default.yaml"
 
 # Load project-specific config, which includes the shared 
 # configuration and some module-specific config updates
-configfile: "config.yaml"
+configfile: "capture_config.yaml"
 
 
 ##### CONFIGURATION UPDATES #####
 
 
 # Use all samples as a default sample list for each module
-config["lcr-modules"]["_shared"]["samples"] = SAMPLES
-config["lcr-modules"]["starfish"]["samples"] = CAPTURE
+config["lcr-modules"]["_shared"]["samples"] = CAPTURE
 
 ##### MODULE SNAKEFILES #####
 
 
 # Load module-specific snakefiles
-
 include: "../modules/slms_3/1.0/slms_3.smk"
+include: "../modules/sage/1.0/sage.smk"
 include: "../modules/utils/2.1/utils.smk"
 include: "../modules/picard_qc/1.0/picard_qc.smk"
-include: "../modules/salmon/1.1/salmon.smk"
-include: "../modules/star/1.4/star.smk"
 include: "../modules/manta/2.3/manta.smk"
-include: "../modules/vcf2maf/1.2/vcf2maf.smk"
+#include: "../modules/vcf2maf/1.2/vcf2maf.smk"
```

Comment:
Rather than commenting out things which are used conditionally you can set them up to only be loaded when necessary. This way the user will not have to dig through the code to find the needed dependency for their use case.